ironic/drivers/modules/drac/management.py

```diff
@@ -357,3 +358,32 @@ def get_sensors_data(self, task):
                   sensor type, which can be processed by Ceilometer.
         """
         raise NotImplementedError()
+
+    @METRICS.timer('DracManagement.reset_idrac')
+    @base.clean_step(priority=CONF.drac.clean_priority_reset_idrac)
```

Comment:
I think the priority for this step should be 0

Reply:
Done